libs/designsystem/src/lib/components/card/card-header/card-header.component.ts

```diff
@@ -9,8 +9,15 @@ export class CardHeaderComponent implements OnInit {
   @Input() title: string;
   @Input() subtitle: string;
   @Input() isTitleBold: boolean;
+  @Input() flagged: 'success' | 'warning' | 'danger' | 'informational' = null;
```

Comment:
Consider creating a type for this string literal (and/or converting to enum type)